ttkwidgets/__init__.py

```diff
@@ -11,3 +11,4 @@
 from ttkwidgets.timeline import TimeLine
 from ttkwidgets.tickscale import TickScale
 from ttkwidgets.table import Table
+from ttkwidgets.validated_entries import *
```

Comment:
Could you change either this or the `ttkwidgets/validated_entries/__init__.py` so that only the classes defined in the `validators.py` module are imported?

Reply:
Can I ask why. I'd argue that the numbers and strings modules are more important since they define actual ready to use entries. If a dev wants to make a custom validator, he can import it with `from ttkwidgets.validated_entries.validators import *` right ?

Reply:
While that is true, it introduces name shadowing, possible import loops (very unlikely in this case) and messes with editors that optimize imports. It hides behaviour that the programmer should decide what to do with, making code harder to debug. Especially because it's standard library modules, I really think it's better not to use an `import *`.